packages/fast-web-utilities/README.md

```diff
@@ -158,3 +158,19 @@ const hyphenatedToPascal = pascalCase("my-string");
 const uppercaseToPascal = pascalCase("MY STRING");
 const whitespaceToPascal = pascalCase(" my string ");
 ```
+
+#### classNames
+A utility for merging class names into a single string conditionally. Accepts any number of strings, functions that return strings and two index arrays where the first index is a string or function that returns a string, and the second index is a boolean.
```

Comment:
This is a little hard to read, can we change it to:

A utility for merging class names into a single string. It can accept any number of arguments which can be any of the following:
- A string
- A functions that returns a string
- An array with two indexes, where the first index is a string or function that returns a string and the second is a boolean that when true adds the resolved first index string to the class names